include/kodlab_mjbots_sdk/robot_base.h

```diff
@@ -50,7 +50,10 @@ namespace kodlab
                 joints.push_back(j); // Copy constructed, uses an implicit upcasting
                 positions_.push_back( j->get_position_reference() );
                 velocities_.push_back( j->get_velocity_reference() );
-                torque_cmd_.push_back( j->get_servo_torque_reference() ); 
+                torque_cmd_.push_back( j->get_servo_torque_reference() );
+                if(j->get_name() != ""){
+                  joint_name_to_index_.insert({j->get_name(), joints.size()-1});
```

Comment:
Might be worth checking if the name already exists in `joint_name_to_index_` and warning about duplicate naming if it does.

Reply:
Map insert fails due to a duplicate entry

Reply:
Fails but does not output any warning.  I meant to add a check similar to `ok` in the [example here](https://en.cppreference.com/w/cpp/container/unordered_map/insert).  Either way, NBD.

Reply:
Yeah. I'm adding a LOG_ERROR in a follow up pr for duplicate names. I'll hold off on merging it in since I should actually test it.